app/src/main/java/live/hms/app2/ui/meeting/pinnedvideo/PinnedVideoFragment.kt

```diff
@@ -151,20 +153,19 @@ class PinnedVideoFragment : Fragment() {
 
     crashlyticsLog(TAG, "Changing pin-view video to $track (previous=$pinnedTrack)")
 
-    val view = SurfaceViewRenderer(context)
+    val view = HMSVideoView(requireContext())
 
     view.apply {
       binding.pinVideo.surfaceViewHolder.forEach {
-        if (it is SurfaceViewRenderer){
-          pinnedTrack?.let { pin-> SurfaceViewRendererUtil.unbind(it, pin)  }
+        if (it is HMSVideoView){
+          it.removeTrack()
         }
       }
       binding.pinVideo.surfaceViewHolder.removeAllViews()
       binding.pinVideo.surfaceViewHolder.addView(this)
       visibility = View.GONE
-        SurfaceViewRendererUtil.bind(view, track).let { success ->
-          if (success) visibility = if (track.video?.isDegraded == true) View.INVISIBLE else View.VISIBLE
-        }
+      track.video?.let { view.addTrack(it) }
+      visibility = if (track.video?.isDegraded == true) View.INVISIBLE else View.VISIBLE
```

Comment:
`visibility = if (track.video?.isDegraded == true) View.INVISIBLE else View.VISIBLE` shuold be called only if  `track.video` is not null